coralproject/sponge
# Issue#66

## What does this PR do?

Refactor the strategy to overwrite credentials via environment variables. It also rename some stuff for clarity. It also fixes how it was getting the foreign entity's name.

## How do I test this PR?

Set environment variables as i env.list and check that they are overwritten.

@coralproject/backend
